src/main.cpp

```diff
@@ -2627,10 +2623,10 @@ static bool ActivateBestChainStep(CValidationState &state, CBlockIndex *pindexMo
         pindexIter = pindexIter->pprev;
     }
     nHeight = nTargetHeight;
-
+    auto ver = libzcash::ProofVerifier::Disabled();
```

Comment:
I think this is left over from testing? Because with this here, all verification pathways from `ProcessNewBlock` are disabled.